### ISSUE TITLE
validation set and loss in tensorboard graph

### ISSUE THREAD

**Issue by joaoalves10** [AntonMu/TrainYourOwnYOLO#250]
Hi @AntonMu i would like to ask you where i can find the graph related to the validation set? When i use the tensorboard only appears the loss of the trainning.

Another question is related to the loss graph from tensorboard. Can you explain to my why the loss has a such a steep decline between the first trainning epochs and the second ones?

I have another question, why do you freeze some layers in the first set of trainninf and then unfreeze them all in the last part? And how many layers do you freeze initially?

![image](https://user-images.githubusercontent.com/89699717/164201949-7d07170c-083e-4a6d-b4a4-afbf3441057f.png)

**Comment by AntonMu** [comment]
Everything you are asking, you can see in the colab: https://github.com/AntonMu/TrainYourOwnYOLO#google-colab-tutorial- 

Hope this helps!

**Comment by joaoalves10** [comment]
it helps a bit. however i do not find the answer to why the loss has a such a steep decline between the first trainning epochs and the second ones?
And why do you freeze some layers in the first set of trainninf and then unfreeze them all in the last part? And how many layers do you freeze initially?

**Comment by AntonMu** [comment]
If you look at the log messages under `!python Train_YOLO.py`, you see the exact numbers of frozen layers in each step. 

Of course, if you freeze layers, you train only a small part of the model, and thus the loss is larger. When you unfreeze all layers, the model has more parameters to learn, and thus the loss is smaller.

**Comment by joaoalves10** [comment]
Thank you so much!



Em seg., 25 de abr. de 2022 às 05:07, AntonMu ***@***.***>
escreveu:
